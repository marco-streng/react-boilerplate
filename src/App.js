// react-hot-loader has to be imported before react and react-dom
import { hot } from 'react-hot-loader/root'
import React from 'react'

import Styled from './Styled'
import Header from './components/Header'

const App = () => (
  <Styled>
    <Header />
  </Styled>
)

export default hot(App)
