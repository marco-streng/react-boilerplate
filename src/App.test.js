import React from 'react'
import renderer from './helper/tests/jest'

import App from './App'

describe('App', () => {
  test('default', () => {
    const tree = renderer(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
