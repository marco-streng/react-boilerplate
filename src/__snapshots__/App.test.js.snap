// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App default 1`] = `
Array [
  <header
    className="Header-sc-1us8icj-0 gSTpjM"
  >
    React Boilerplate
  </header>,
  <img
    alt="Boilerplate"
    className="App__Logo-sc-6uwloo-0 bAwzNB"
    src={Object {}}
  />,
  <div
    className="App__Center-sc-6uwloo-1 crVkKA"
  >
    <p>
      React ❤️styled component ❤️Jest ❤️webpack ❤️ JavaScript Standard Style ❤️stylelint
    </p>
    <a
      className="LinkButton-sc-16kn36j-0 bgDtbx"
      href="https://github.com/marco-streng/react-boilerplate"
      target="_blank"
    >
      GitHub
    </a>
  </div>,
]
`;
